### ISSUE TITLE
Incorrect Channel Numbers for Frequencies

### ISSUE THREAD

**Issue by christianrowlands** [wiglenet/wigle-wifi-wardriving#417]
Looking through the code, I happened to notice that a couple of the channel number might be wrong for the corresponding frequency mapping:

I think these two:
```
freqToChanTemp.put(5300, 58);
freqToChanTemp.put(5320, 60);
```

Should be:
```
freqToChanTemp.put(5300, 60);
freqToChanTemp.put(5320, 64);
```

https://github.com/wiglenet/wigle-wifi-wardriving/blob/bf63f31c8256b149a68bec87ed7da934b71222e2/wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/model/Network.java#L70-L71

There might be others that need updating as well as I did not check all of them.

**Comment by christianrowlands** [comment]
It also looks like 5570 needs to be changed to 5580:

![image](https://user-images.githubusercontent.com/8020686/81981467-52b44880-95fe-11ea-9db5-e710f801c997.png)

I have a branch created with the fix, but don't have permissions to push the branch to open a PR.

**Comment by bobzilladev** [comment]
Looks like you're right! You could fork this repo, commit the diff there, then submit a PR from the fork if you want. We should probably add all the interstitial channels too at some point.
https://en.wikipedia.org/wiki/List_of_WLAN_channels

**Comment by christianrowlands** [comment]
https://github.com/wiglenet/wigle-wifi-wardriving/pull/418/files

**Comment by rksh** [comment]
fix will be included in release 2.54 - thank you for your help!
